### ISSUE TITLE
string to Color ?

### ISSUE THREAD

**Issue by c0ncentus** [ilariaventurini/colors-convert#19]
Hi; 

I read the doc but ... none function about construct Color with a string ......

**Comment by ilariaventurini** [comment]
Hi, thank you for the suggest!

**Comment by ilariaventurini** [comment]
Hey c0ncentus, I just released version 1.4.0 that includes the `stringToColor` method you need.

You can use it like this:

```js
stringToColor('#ffF000') // '#FFF000'
stringToColor('rgb(0, 0, 0)') // { r: 0, g: 0, b: 0}
stringToColor('rgba(0, 0, 0, 1)') // { r: 0, g: 0, b: 0, a: 1}
stringToColor('cmyk(0, 0, 0, 0)') // { c: 0, m: 0, y: 0, k: 0}
stringToColor('hsl(0, 0%, 0%)') // { h: 0, s: 0, l: 0}
stringToColor('hsla(0, 0%, 0%, 0)') // { h: 0, s: 0, l: 0, a: 0}
```
